detection_mode: {
  marker: 7, 
  circle: 12
}